### PR TITLE
Create first timers issue template.md

### PR DESCRIPTION
**Pull request by faithngetich**
Issue [link](https://github.com/publiclab/plots2/issues/3681#issuecomment-429254015)

### CODE REVIEW

File: .github/first-timers-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 
+
+If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!
+
+We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
+
+If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!
```
Comment (jywarren) [inline .github/first-timers-issue-template.md]:
```suggestion
If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/leaflet-environmental-layers/labels/help-wanted) issues. Thanks!
```

File: .github/first-timers-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
```
Comment (jywarren) [inline .github/first-timers-issue-template.md]:
```suggestion
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 
```

File: .github/first-timers-issue-template.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
+
+### 📋 Step by Step
+
+- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
+
+- [ ] 📝 **Update** the file [$FILENAME]($BRANCH_URL) in the `$REPO` repository (press the little pen Icon) and edit the line as shown below. 
+
+[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!
```
Comment (jywarren) [inline .github/first-timers-issue-template.md]:
```suggestion
[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!
```

**Comment by faithngetich** [comment]
Hi @jywarren I've made the changes.

**Comment by jywarren** [comment]
Thank you!!!!

**Pull request merged**